src/util/update.ts

```diff
@@ -0,0 +1,6 @@
+import {User} from "../entity/User"
+
+export async function updateName(discordID : string, name : string)
+{
+    await User.update({discordId : discordID}, {username : name}) //Updates username field wrt DiscordID
```

Comment:
You can use the object shorthand notation here for `discordId`.

Reply:
Okay